challenges/qr_code_component/tailwind.config.js

```diff
@@ -3,8 +3,11 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        'optimum-blue': '#009efc',
-        'optimum-darkblue': '#0389e1',
```

Comment:
You've deleted colors we're using for the links. Search for "text-optimum-blue".